### ISSUE TITLE
How Can I Move Items Between Memory Cards/or From Memory Card to Handy Chest?

### ISSUE THREAD

**Issue by D3Gamer** [maruohon/enderutilities#102]
I didn't know where else to put this on this website...I don't know if this is the right place?

**How do you move items between memory cards/or from memory card to handy chest??**

I've been unable to find any explanation on the internet on how to move items from memory card to memory card. Playing Hypixel Skyblock has gotten me into the habit of keeping everything I pickup...  and It's difficult moving 4,096 cobblestone from inventory to memory card in handy chest without having an instant way to do it. There must be a way.. right??

**Comment by maruohon** [comment]
There is no direct way. The Handy Chest does use the cards, so you could just put the card in a Handy Chest. But assuming you just want to pull certain items from it, not leave the entire card there, I guess your best option is to use a pipe of some kind to filter and move the items you want to another Handy Chest (or another large storage, such as the MSU or whatever I called them).

You can use for example the Filtered Inserter to pull items from one Handy Chest to another, at a rate of one stack per game tick.

**Comment by D3Gamer** [comment]
Oh okay thanks. Is the filtered inserter a newer thing from this mod? I use 1.10 and I don't know if there is an updated version of this mod for 1.10.

**Comment by maruohon** [comment]
Looks like it was added in version 0.7.0, so it does also exist in the latest 1.10.2 versions of the mod.

**Comment by D3Gamer** [comment]
For some reason 1.10.2 mods will not work in regular 1.10 for me

**Comment by maruohon** [comment]
Oh you are actually in 1.10.0... is there a particular reason for that, did some of the mods you are using not update to 1.10.2? Looks like it might be just a couple of vanilla methods that changed between 1.10 and 1.10.2 that prevent the 1.10.2 Ender Utilites versions from working on 1.10, although it may also depend on a newer JEI version, I'm not sure.

**Comment by D3Gamer** [comment]
Oh wow really?.. I guess I never bothered updating my mods any further than 1.10. I'm not using any particular modpack of any kind I'm just using around 55 mods that i've picked out that I like. I guess I should look into updating my game to 1.10.2 or perhaps 1.12.2. I think some of these mods didn't update though I don't know.. it wouldn't surprise me.

I've had most of these mods, at least 30 of them, since 2016 and I'm just now getting into them and how they work. I've had ender utilities for that long and I never realized the handy bag was a thing until recently. But if you say i'm missing out on half of the content of ender utilities I'd better get my game upgraded so I can play that. 
 If I get all of the mods I currently am using to 1.12.2 the world I am using now should work, right?

Thank you for answering all of my questions, I don't want to bother you.

**Comment by maruohon** [comment]
There may be changes between Minecraft versions so that some blocks or items can vanish, if the mod went through some major changes during the version update. I'd say it's rather unlikely that mods would not have updated from 1.10 to 1.10.2, or that something major would break in that update. But from 1.10.x to 1.12.2 it's a lot more possible.

**Always** make a full back of both the world save, and also your configs and mods. Basically backup everything (the entire `.minecraft` directory) before updating. Or maybe rather assemble the same mods again in the later MC version in a different pack/instance and then just copy over the config folder and the saves before starting it the first time.

**Comment by D3Gamer** [comment]
Okay, thank you very much.

**Comment by DrZed** [comment]
Don't know if it was mentioned but you can middle click a slot and middle click the empty slot in another tab that has another card it moves the whole stack up to the size limit of the other card.
